src/Component/RuleFieldContainer/RuleFieldContainer.less

```diff
@@ -1,6 +1,6 @@
 .gs-rule-field-container {
   display: flex;
-  div:first-child {
+  >div:first-child {
```

Comment:
cant we just use our own class / selector here which is more stable than this one?

Reply:
Done in https://github.com/geostyler/geostyler/pull/1909/commits/e3de4cffa5dcfd1ce6ce7cc8cfe13b24fb95dc27